docs/source/conf.py

```diff
@@ -0,0 +1,83 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../.."))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "AugLy"
+copyright = "2021, Joanna Bitton and Zoe Papakipos"
+author = "Joanna Bitton and Zoe Papakipos"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.10"
```

Comment:
Will we have to update this too? In that case it might be worth making a text file in the repo that contains the latest version number, and we just update that when updating the package & read from the text file both here & in `setup.py`.

Reply:
@zpapakipos Yep that would be manual -- the text file should be more efficient.